### DISCUSSION TITLE
Cascaded balancers (Neey)

### DISCUSSION THREAD

**Discussion by FusionStorage-DF** [syssi/esphome-jk-bms#563] [General]
Hi is it possible to set this op for cascaded balanceres out of the box?
min/max, voltage etc. of the hole stack

B.R.

**Comment by syssi** [comment]
I don't understand your question. Could you describe your idea / setup / use case?

**Comment by FusionStorage-DF** [comment]
it is so you can have multible balanceres in series when you have more then 24 cells.

so 2 cells is overlapping, 

It would be nice if it can show as one pack, insted of multible balancers?

![neey](https://github.com/user-attachments/assets/67e60f26-c121-4699-9906-af1adf825d7b)

**Reply by syssi** [reply]
Got it. You can use this project to retrieve all the measurements from two device but you have to calculate the min/max voltage over all cell voltages on your own (f.e. using a template sensor + lambda).

**Comment by FusionStorage-DF** [comment]
Ok Thanks! New in this esphome stuff, so trying to understand how it works ;)

**Reply by syssi** [reply]
Ping me if you need some guidance. The first step will be to talk to both balancers using a single ESP32.

**Reply by FusionStorage-DF** [reply]
Thank you Sir :) Do you know if the esp32-s3 is still limited to 3 bt devices? the spec says it should be able to have 9 connections?

**Reply by syssi** [reply]
At the moment there is a hard limit in ESPHome so you aren't able to create more BLE client instances: https://github.com/esphome/esphome/pull/7088

In general it's a soft limit and depends on available resources.
